src/OpenTelemetry.Exporter.Console/ConsoleActivityJsonExporter.cs

```diff
@@ -0,0 +1,160 @@
+using System.Diagnostics;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Exporter
+{
+    public class ConsoleActivityJsonExporter : ConsoleExporter<Activity>
```

Comment:
@brunomartinspro What's the motivation behind adding this new exporter to the repo? `ConsoleExporter` is a simple lightweight exporter meant for quick and naive testing/ troubleshooting for the developers. It was designed to be simple.

Reply:
@cijothomas Why do we want to add this to the existing exporter though? Users are allowed to create their own custom exporters and use them.

Reply:
Its good to support a more standard/predictable output in the consoleexporter, without forcing everyone to author their own exporter.
The predictable/standard output could be the https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/specification.md#json-protobuf-encoding OR could be the Prometheus exposition format for Metrics (there were issues asking for these earlier (this capability existed long back, but got removed), can't find it with my quick search)

Reply:
> Its good to support a more standard/predictable output in the consoleexporter, without forcing everyone to author their own exporter.

I definitely see value in standardizing the ConsoleExporter output. If we decide to go forward with this though, I think the implementation should meet certain requirements:
- We want to keep ConsoleExporter simple, so I don't think we should make this an optional feature. Instead, we should simply update the existing behavior to output data in the standardized format
- ConsoleExporter should not take any additional dependencies or bump the version of existing dependencies that might cause the user to run into version conflict issues with their application code

Reply:
Agree with this.
It would also be good to add disclaimer that the output format could change (it has changed many times in the past) and people should not take dependency on it. (i.e use it for learning purposes, but not for production purposes to have telemetry output to stdout, and have some agent parse it to store telemetry)